content/doc/book/pipeline/index.adoc

```diff
@@ -6,43 +6,138 @@ layout: chapter
 :author:
 :email: jenkinsci-users@googlegroups.com
 :sectanchors:
-:toc: left
+:toc:
 
 = Pipeline
 
-This chapter will cover all aspects of the Jenkins Pipeline, from running pipeline jobs
-to writing your own pipeline code.
+This chapter will cover all aspects of Jenkins Pipeline, from running pipeline jobs
+to writing your own pipeline code, and even extending Pipeline.
 
 This chapter is intended to be used by Jenkins users of all skill levels,
 but beginners may need to refer to some sections of "<<using#,Using Jenkins>>"
 to understand some topics covered in this chapter.
 
-The sections are ordered from simpler to progressively more complex subjects.
-The earlier sections are intended for those new to pipeline or unfamiliar with its
-latest features.  The later sections with discuss expert-level considerations
-and corner-cases.
```

Comment:
This is intended as a note to contributors.  Could we discuss your objections/thoughts?